### PR TITLE
fix(global): DOMA-10345 format eror translations

### PR DESCRIPTION
**Pull request by YEgorLu**
Fixed error messages for user formats according to #ADR6 in condo and apps. Needed for pahaz in [PR](https://github.com/open-condo-software/condo/pull/5243)

### CODE REVIEW

**Review by Alllex202 — APPROVED** [review]

**Review by pahaz — APPROVED** [review]

File: apps/condo/domains/meter/schema/RegisterMetersReadingsService.js [hunk 1]
```diff
@@ -40,13 +40,12 @@ const DATE_PARSING_FORMATS = [
 const READINGS_LIMIT = 500
 
 const ERRORS = {
-    TOO_MUCH_READINGS: (sentCount) => ({
+    TOO_MUCH_READINGS: {
```
Comment (abshnko) [inline apps/condo/domains/meter/schema/RegisterMetersReadingsService.js:43]:
```suggestion
    TOO_MANY_READINGS: {
```

File: apps/condo/domains/meter/schema/RegisterMetersReadingsService.js [hunk 2]
```diff
@@ -40,13 +40,12 @@ const DATE_PARSING_FORMATS = [
...
         code: BAD_USER_INPUT,
         type: TOO_MUCH_READINGS,
         message: `Too much readings. Maximum is ${READINGS_LIMIT}.`,
         messageForUser: 'api.meter.registerMetersReadings.TOO_MUCH_READINGS',
```
Comment (abshnko) [inline apps/condo/domains/meter/schema/RegisterMetersReadingsService.js:47]:
```suggestion
        messageForUser: 'api.meter.registerMetersReadings.TOO_MANY_READINGS',
```

File: apps/condo/domains/meter/schema/RegisterMetersReadingsService.js [hunk 3]
```diff
@@ -263,7 +259,10 @@ const RegisterMetersReadingsService = new GQLCustomSchema('RegisterMetersReading
                 const { data: { dv, sender, organization, readings } } = args
 
                 if (readings.length > READINGS_LIMIT) {
-                    throw new GQLError(ERRORS.TOO_MUCH_READINGS(readings.length), context)
+                    throw new GQLError({
+                        ...ERRORS.TOO_MUCH_READINGS,
```
Comment (abshnko) [inline apps/condo/domains/meter/schema/RegisterMetersReadingsService.js:263]:
```suggestion
                        ...ERRORS.TOO_MANY_READINGS,
```

File: apps/condo/domains/acquiring/schema/CreatePaymentByLinkService.js [hunk 4]
```diff
@@ -36,7 +36,7 @@ const ERRORS = {
         code: BAD_USER_INPUT,
         type: ALREADY_EXISTS_ERROR,
         message: 'Provided receipt already paid',
-        messageForUser: 'api.billing.error.alreadyPaid',
+        messageForUser: 'api.billing.billingReceipt.RECEIPT_ALREADY_PAID_ERROR',
```
Comment (abshnko) [inline apps/condo/domains/acquiring/schema/CreatePaymentByLinkService.js:39]:
```suggestion
        messageForUser: 'api.billing. createPaymentByLink.RECEIPT_ALREADY_PAID_ERROR',
```

seems like it's appropriate to use the mutation name, not the model

**Review by abshnko — APPROVED** [review]

File: apps/condo/domains/meter/schema/RegisterMetersReadingsService.js (hunk 3 above)
Comment (abshnko) [reply apps/condo/domains/meter/schema/RegisterMetersReadingsService.js:263]: ..and in other places where it's used

File: apps/condo/domains/acquiring/schema/CreatePaymentByLinkService.js (hunk 4 above)
Comment (YEgorLu) [reply apps/condo/domains/acquiring/schema/CreatePaymentByLinkService.js:39]: this error translation is being used in different apps and two mutation in condo, so i think i'ts easier to write 'billingReceipt', than determine which mutation is more correct for this error

File: apps/condo/domains/meter/schema/RegisterMetersReadingsService.js (hunk 2 above)
Comment (YEgorLu) [reply apps/condo/domains/meter/schema/RegisterMetersReadingsService.js:47]: too much to do it in all places in this pr

File: apps/condo/domains/meter/schema/RegisterMetersReadingsService.js (hunk 3 above)
Comment (YEgorLu) [reply apps/condo/domains/meter/schema/RegisterMetersReadingsService.js:263]: too much to do it in all places in this pr

File: apps/condo/domains/meter/schema/RegisterMetersReadingsService.js (hunk 1 above)
Comment (YEgorLu) [reply apps/condo/domains/meter/schema/RegisterMetersReadingsService.js:43]: too much to do it in all places in this pr

**Pull request merged**